code/game/machinery/stasis.dm

```diff
@@ -7,7 +7,6 @@
 	density = FALSE
 	can_buckle = TRUE
 	buckle_lying = 90
-	circuit = /obj/item/circuitboard/machine/stasis
```

Comment:
You mentioned this in the comments but not the PR body.

Reply:
docced